pkg/minishift/network/settings_windows.go

```diff
@@ -41,10 +42,21 @@ var (
 )
 
 func isUsingDefaultSwitch() bool {
-	switchEnvName := "HYPERV_VIRTUAL_SWITCH"
-	switchEnvValue := os.Getenv(switchEnvName)
+	posh := powershell.New()
+	switchIdCommand := `(Get-VMSwitch -Name $swname).Id.Guid`
+	getSwitchIdForVmCommand := fmt.Sprintf("$swname=(Get-VM \"%s\" | Get-VMNetworkAdapter | select SwitchName).SwitchName;%s",
+		profile.GetActiveProfile(),
```

Comment:
We might end up jumbling arund here with the active profile. but for now, this works, although I believe this might get risky going forward...